src/lib/locale.store.ts

```diff
@@ -1,42 +1,68 @@
 import { readable, writable, type Readable, type Writable } from 'svelte/store';
 import { z } from 'zod';
 
-import * as fallback from '$locales/us/en/index.json';
+import fallback from '$locales/us/en/index.json';
+
+export const stringsSchema = z
+  .object({
+    alreadyHaveAnAccount: z.string(),
+    dontHaveAnAccount: z.string(),
+    closeModal: z.string(),
+    chooseDifferentUsername: z.string(),
+    constraintViolationForPassword: z.string(),
+    constraintViolationForValue: z.string(),
+    customSecurityQuestion: z.string(),
+    doesNotMeetMinimumCharacterLength: z.string(),
+    ensurePasswordIsMoreThan: z.string(),
+    ensurePasswordHasOne: z.string(),
+    exceedsMaximumCharacterLength: z.string(),
+    fieldCanNotContainFollowingCharacters: z.string(),
+    fieldCanNotContainFollowingValues: z.string(),
+    givenName: z.string(),
+    inputRequiredError: z.string(),
+    loading: z.string(),
+    loginButton: z.string(),
+    loginFailure: z.string(),
+    loginHeader: z.string(),
+    loginSuccess: z.string(),
+    mail: z.string(),
+    minimumNumberOfNumbers: z.string(),
+    minimumNumberOfLowercase: z.string(),
+    minimumNumberOfUppercase: z.string(),
+    minimumNumberOfSymbols: z.string(),
+    nameCallback: z.string(),
+    nextButton: z.string(),
+    notToExceedMaximumCharacterLength: z.string(),
+    noLessThanMinimumCharacterLength: z.string(),
+    passwordCallback: z.string(),
+    passwordRequirements: z.string(),
+    pleaseCheckValue: z.string(),
+    preferencesMarketing: z.string(),
+    preferencesUpdates: z.string(),
+    provideCustomQuestion: z.string(),
+    registerButton: z.string(),
+    registerHeader: z.string(),
+    registerSuccess: z.string(),
+    requiredField: z.string(),
+    securityAnswer: z.string(),
+    securityQuestions: z.string(),
+    securityQuestionsPrompt: z.string(),
+    showPassword: z.string(),
+    sn: z.string(),
+    submitButton: z.string(),
+    successMessage: z.string(),
+    termsAndConditions: z.string(),
+    tryAgain: z.string(),
+    useValidEmail: z.string(),
+    unrecoverableError: z.string(),
+    unknownNetworkError: z.string(),
+    usernameRequirements: z.string(),
+    validatedCreatePasswordCallback: z.string(),
+    validatedCreateUsernameCallback: z.string(),
+    valueRequirements: z.string(),
+  })
+  .strict();
```

Comment:
Adding this `strict` property, it will report unknown properties as well, not just ignore them. This is good for accidentally misspelling a prop.

Reply:
Maybe I misunderstand what we are doing here (see my other comment on Choice Callback), but do we not want to allow additional properties, in case they occur in a node on the server (which is detached from this)?

Reply:
Currently, this object exactly lines up with the locale files that we have in the project. Even if the developer wanted to add a property, it wouldn't result in any content lookup because the content file wouldn't have the key. It would then fallback to the provided text from AM.

This is a problem that we will really need to think about, but I'm unsure how to solve it without real world testing and feedback. We will likely iterate on how content is handled in this framework multiple times in the coming months.